src/components/ContactList.tsx

```diff
@@ -29,7 +29,7 @@ const ContactList = () => {
     const [searchParams, setSearchParams] = useSearchParams();
     
     // console.log("searchInput is: ", searchInput);
```

Comment:
Looking good
Nit: probably worth removing all the old commented-out console logs